Mathlib/SetTheory/Ordinal/Arithmetic.lean

```diff
@@ -1184,6 +1193,12 @@ theorem bddAbove_iff_small {s : Set Ordinal.{u}} : BddAbove s ↔ Small.{u} s :=
   ⟨fun ⟨a, h⟩ => small_subset <| show s ⊆ Iic a from fun _ hx => h hx, fun _ =>
     bddAbove_of_small _⟩
 
+theorem bddAbove_range_comp {ι : Type u} {f : ι → Ordinal.{v}} (hf : BddAbove (range f))
+    (g : Ordinal.{v} → Ordinal.{max v w}) : BddAbove (range (g ∘ f)) := by
+  rw [range_comp, bddAbove_iff_small]
+  rw [bddAbove_iff_small] at hf
+  exact small_lift _
```

Comment:
I'm not sure how this proof works! Is there something like "`BddAbove s → BddAbove (g '' s)` to extract from it?

Reply:
Sure, I've added `bddAbove_image`.